dev-lang/gnucobol/gnucobol-2.2.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="an open-source COBOL compiler"
+HOMEPAGE="https://open-cobol.sourceforge.io"
+SRC_URI="mirror://sourceforge/open-cobol/${P}.tar.gz"
+
+LICENSE="GPL-3+ LGPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
```

Comment:
I am suspicious about retaining ppc [keywords](https://devmanual.gentoo.org/keywording/index.html) given how major of a change this is.